main.tf

```diff
@@ -0,0 +1,12 @@
+
+resource "null_resource" "ansible_provisioner" {
```

Comment:
Since this is inside the `tf_ansible` module, we don't need to prefix the resources with `ansible_`